opendebates/models.py

```diff
@@ -50,6 +50,10 @@ def user_display_name(self):
 
     editors_pick = models.BooleanField(default=False)
     approved = models.BooleanField(default=False, db_index=True)
+
+    # if True, will not show up again in moderation list.
+    moderated_removal = models.BooleanField(default=False, db_index=True)
```

Comment:
I'm not sure I understand what this field does. Is it a permanent indication that no matter how many times this question is flagged, moderators will never see it again?  And if so, I'm not sure of the use case here.